ee/clickhouse/queries/trends/total_volume.py

```diff
@@ -64,10 +63,10 @@ def _total_volume_query(self, entity: Entity, filter: Filter, team_id: int) -> T
                 content_sql = VOLUME_SQL.format(event_query=event_query, **content_sql_params)
 
             null_sql = NULL_SQL.format(
```

Comment:
Nitpicking: Parameters > interpolation here - reduces risk of injection.

Reply:
I reviewed the things going into this to make sure they are sanitized (incl. added checks in `filter.interval`) but sure, moved values to `params`.

Reply:
Reason I'm nitpicking here is that if we refactor things we don't review all queries where a value is interpolated into -> could cause an issue then :)

It's a probabilities game, with each interpolation raising the odds of eventually making a mistake.